README

```diff
@@ -265,6 +265,11 @@ Documentation
 
 Documentation is available at https://elasticsearch-dsl.readthedocs.io.
 
+Contribution Guide
+------------------
+
+Want to hack on Elasticsearch DSL? Awesome! We have `Contribution-Guide <https://github.com/elastic/elasticsearch-dsl-py/blob/master/CONTRIBUTING.md>`_.
```

Comment:
You have included the `contributing` chapter in the docs themselves, in that case this should point to that page in the docs, not to CONTRIBUTING document on github, via a direct link - http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-documents